22-02-2020 20:20:41.799 [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker.postProcessAfterInitialization - Bean 'configurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$f51faf0] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
22-02-2020 20:20:44.750 [main] INFO  com.ims.InventorySystemApplication.logStartupProfileInfo - No active profile set, falling back to default profiles: default
22-02-2020 20:20:46.894 [main] INFO  o.s.cloud.context.scope.GenericScope.setSerializationId - BeanFactory id=784b9b19-4f44-37e7-9815-0504d43263b6
22-02-2020 20:20:47.136 [main] INFO  o.s.c.s.PostProcessorRegistrationDelegate$BeanPostProcessorChecker.postProcessAfterInitialization - Bean 'org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration' of type [org.springframework.cloud.autoconfigure.ConfigurationPropertiesRebinderAutoConfiguration$$EnhancerBySpringCGLIB$$f51faf0] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
22-02-2020 20:20:49.204 [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer.initialize - Tomcat initialized with port(s): 8084 (http)
22-02-2020 20:20:49.240 [main] INFO  o.a.coyote.http11.Http11NioProtocol.log - Initializing ProtocolHandler ["http-nio-8084"]
22-02-2020 20:20:49.301 [main] INFO  o.a.catalina.core.StandardService.log - Starting service [Tomcat]
22-02-2020 20:20:49.302 [main] INFO  o.a.catalina.core.StandardEngine.log - Starting Servlet engine: [Apache Tomcat/9.0.17]
22-02-2020 20:20:49.889 [main] INFO  o.a.c.c.C.[Tomcat].[localhost].[/].log - Initializing Spring embedded WebApplicationContext
22-02-2020 20:20:49.889 [main] INFO  o.s.web.context.ContextLoader.prepareWebApplicationContext - Root WebApplicationContext: initialization completed in 5102 ms
22-02-2020 20:20:50.208 [main] WARN  c.n.c.sources.URLConfigurationSource.<init> - No URLs will be polled as dynamic configuration sources.
22-02-2020 20:20:50.208 [main] INFO  c.n.c.sources.URLConfigurationSource.<init> - To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
22-02-2020 20:20:50.224 [main] WARN  c.n.c.sources.URLConfigurationSource.<init> - No URLs will be polled as dynamic configuration sources.
22-02-2020 20:20:50.225 [main] INFO  c.n.c.sources.URLConfigurationSource.<init> - To enable URLs as dynamic configuration sources, define System property archaius.configurationSource.additionalUrls or make config.properties available on classpath.
22-02-2020 20:20:50.721 [main] INFO  o.s.s.c.ThreadPoolTaskExecutor.initialize - Initializing ExecutorService 'applicationTaskExecutor'
22-02-2020 20:20:57.077 [main] INFO  o.s.c.n.eureka.InstanceInfoFactory.create - Setting initial instance status as: STARTING
22-02-2020 20:20:57.195 [main] INFO  c.netflix.discovery.DiscoveryClient.<init> - Initializing Eureka in region us-east-1
22-02-2020 20:20:57.198 [main] INFO  c.netflix.discovery.DiscoveryClient.<init> - Client configured to neither register nor query for data.
22-02-2020 20:20:57.231 [main] INFO  c.netflix.discovery.DiscoveryClient.<init> - Discovery Client initialized at timestamp 1582383057227 with initial instances count: 0
22-02-2020 20:20:57.243 [main] INFO  o.s.c.n.e.s.EurekaServiceRegistry.register - Registering application TEST-SERVICE with eureka with status UP
22-02-2020 20:20:57.289 [main] INFO  o.a.coyote.http11.Http11NioProtocol.log - Starting ProtocolHandler ["http-nio-8084"]
22-02-2020 20:20:57.374 [main] INFO  o.s.b.w.e.tomcat.TomcatWebServer.start - Tomcat started on port(s): 8084 (http) with context path ''
22-02-2020 20:20:57.390 [main] INFO  com.ims.InventorySystemApplication.logStarted - Started InventorySystemApplication in 20.322 seconds (JVM running for 22.142)
22-02-2020 20:20:57.423 [main] INFO  com.ims.InventorySystemApplication.main - ________________________________________________________________________________________________________________________________
   ____      ___     _       ___     __   __ __    __   __ _ _ 
/*/        /*/ \*\  |*|    /*/ \*\  \*\  /*/ \*\  /*/   \ \ \ \
|*|   ___ |*|___|*| |*|   |*|___|*|  \*\/*/   \*\/*/     \ \ \ \ 
|*|   |*| |*|   |*| |*|   |*|   |*|  /*/\*\    |*|       ) ) ) )
\*\---|*| |*|   |*| |*|__ |*|   |*| /*/  \*\   |*|      / / / /
 `------' '-'   '-' '----''-'   '-''-'    '-'  '-'     /_/_/_/
____________________________________________________________________________________________________________________________
22-02-2020 20:20:57.425 [main] INFO  com.ims.InventorySystemApplication.main - LG220220082036_409663969::REDIS CONFIGURATION START
22-02-2020 20:20:57.428 [main] INFO  com.ims.InventorySystemApplication.isRedisServerUp - LG220220082036_409663969::Checking Redis server is up & Running on port no: 6379
22-02-2020 20:20:59.525 [main] INFO  com.ims.InventorySystemApplication.isRedisServerUp - LG220220082036_409663969::REDIS SERVER IS DOWN
22-02-2020 20:20:59.526 [main] ERROR com.ims.InventorySystemApplication.isRedisServerUp - LG220220082036_409663969::############EXCEPTION START###########
22-02-2020 20:20:59.526 [main] ERROR com.ims.InventorySystemApplication.isRedisServerUp - LG220220082036_409663969::redis.clients.jedis.exceptions.JedisConnectionException: Could not get a resource from the pool
22-02-2020 20:20:59.526 [main] ERROR com.ims.InventorySystemApplication.isRedisServerUp - LG220220082036_409663969::##########EXCEPTION END###########
22-02-2020 20:20:59.527 [main] INFO  com.ims.InventorySystemApplication.main - LG220220082036_409663969::REDIS CONFIGURATION END
22-02-2020 20:20:59.528 [main] INFO  com.ims.InventorySystemApplication.main - LG220220082036_409663969::QUITING SERVICE
22-02-2020 20:20:59.528 [main] INFO  com.ims.InventorySystemApplication.main - LG220220082036_409663969:: Execution time:  22875
22-02-2020 20:20:59.528 [main] INFO  com.ims.InventorySystemApplication.main - LG220220082036_409663969::XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
22-02-2020 20:23:32.862 [http-nio-8084-exec-1] INFO  o.a.c.c.C.[Tomcat].[localhost].[/].log - Initializing Spring DispatcherServlet 'dispatcherServlet'
22-02-2020 20:23:32.865 [http-nio-8084-exec-1] INFO  o.s.web.servlet.DispatcherServlet.initServletBean - Initializing Servlet 'dispatcherServlet'
22-02-2020 20:23:32.883 [http-nio-8084-exec-1] INFO  o.s.web.servlet.DispatcherServlet.initServletBean - Completed initialization in 17 ms
22-02-2020 20:23:33.000 [http-nio-8084-exec-1] INFO  c.i.c.AreaLocationSourceController.getAreaLocations - 2587[<<] [<<] [<<] [<<] [<<] **NEW REQUEST* [<<] [<<] [<<] [<<] [<<]
22-02-2020 20:23:33.000 [http-nio-8084-exec-1] INFO  c.i.c.AreaLocationSourceController.getAreaLocations - 2587##INPUT START##
22-02-2020 20:23:33.001 [http-nio-8084-exec-1] INFO  c.i.c.AreaLocationSourceController.getAreaLocations - 2587##INPUT END##
22-02-2020 20:23:33.002 [http-nio-8084-exec-1] INFO  c.i.s.AreaLocationSourceServiceImpl.getAreaLocationService - 2587[<<] [<<] [<<] [<<] [<<] **NEW REQUEST* [<<] [<<] [<<] [<<] [<<]
22-02-2020 20:23:33.002 [http-nio-8084-exec-1] INFO  c.i.s.AreaLocationSourceServiceImpl.getAreaLocationService - 2587##INPUT START##
22-02-2020 20:23:33.003 [http-nio-8084-exec-1] INFO  c.i.s.AreaLocationSourceServiceImpl.getAreaLocationService - 2587##INPUT END##
22-02-2020 20:23:33.007 [http-nio-8084-exec-1] INFO  com.ims.helper.HibernateUtil.getSessionFactory - 2587getting session from database
22-02-2020 20:23:33.311 [http-nio-8084-exec-1] INFO  org.hibernate.Version.logVersion - HHH000412: Hibernate Core {5.0.12.Final}
22-02-2020 20:23:33.316 [http-nio-8084-exec-1] INFO  org.hibernate.cfg.Environment.<clinit> - HHH000206: hibernate.properties not found
22-02-2020 20:23:33.320 [http-nio-8084-exec-1] INFO  org.hibernate.cfg.Environment.buildBytecodeProvider - HHH000021: Bytecode provider name : javassist
22-02-2020 20:23:33.346 [http-nio-8084-exec-1] INFO  com.ims.helper.HibernateUtil.getSessionFactory - 2587Trying to connect db URL: jdbc:mysql://localhost:3306/INVENTORY_SYSTEM_DB
22-02-2020 20:23:33.445 [http-nio-8084-exec-1] INFO  o.h.annotations.common.Version.<clinit> - HCANN000001: Hibernate Commons Annotations {5.0.1.Final}
22-02-2020 20:23:33.666 [http-nio-8084-exec-1] WARN  org.hibernate.orm.connections.configure - HHH10001002: Using Hibernate built-in connection pool (not for production use!)
22-02-2020 20:23:33.681 [http-nio-8084-exec-1] INFO  org.hibernate.orm.connections.buildCreator - HHH10001005: using driver [com.mysql.cj.jdbc.Driver] at URL [jdbc:mysql://localhost:3306/INVENTORY_SYSTEM_DB?useSSL=false]
22-02-2020 20:23:33.682 [http-nio-8084-exec-1] INFO  org.hibernate.orm.connections.buildCreator - HHH10001001: Connection properties: {user=root, password=****}
22-02-2020 20:23:33.683 [http-nio-8084-exec-1] INFO  org.hibernate.orm.connections.buildCreator - HHH10001003: Autocommit mode: false
22-02-2020 20:23:33.692 [http-nio-8084-exec-1] INFO  o.h.e.j.c.i.DriverManagerConnectionProviderImpl.<init> - HHH000115: Hibernate connection pool size: 20 (min=1)
22-02-2020 20:23:34.024 [http-nio-8084-exec-1] INFO  org.hibernate.dialect.Dialect.<init> - HHH000400: Using dialect: org.hibernate.dialect.MySQL5Dialect
22-02-2020 20:23:35.911 [http-nio-8084-exec-1] INFO  c.i.s.AreaLocationSourceServiceImpl.getAreaLocationService - 2587 Execution time : 2909
22-02-2020 20:23:35.911 [http-nio-8084-exec-1] INFO  c.i.s.AreaLocationSourceServiceImpl.getAreaLocationService - 2587[>>] [>>] [>>] [>>] [>>] **SENDING RESPONSE TO CLIENT** [>>] [>>] [>>] [>>] [>>] 

22-02-2020 20:23:35.912 [http-nio-8084-exec-1] INFO  c.i.c.AreaLocationSourceController.getAreaLocations - 2587 Execution time : 2914
22-02-2020 20:23:35.912 [http-nio-8084-exec-1] INFO  c.i.c.AreaLocationSourceController.getAreaLocations - 2587[>>] [>>] [>>] [>>] [>>] **SENDING RESPONSE TO CLIENT** [>>] [>>] [>>] [>>] [>>] 

